source/connect/advanced-connection.txt

```diff
@@ -12,3 +12,88 @@ Advanced Connection Tab
    :depth: 1
    :class: singlecol
 
+
+The :guilabel:`Advanced` connection tab provides additional connection options 
+for your deployment. These options can be used alongside the 
+:ref:`General <general-connection-tab>`, :ref:`Authentication 
+<authentication-connection-tab>`, :ref:`TLS / SSL <tls-ssl-tab>`, and 
+:ref:`Proxy / SSH Tunnel <ssh-connection>` connection options. 
+
+Procedure
+---------
+
+.. procedure::
+   :style: normal
+
+   .. step:: Click :guilabel:`Advanced Connection Options`.
+
+      .. figure:: /images/compass/advanced-connection-options.png
+         :figwidth: 690px
+         :alt: New Advanced Connection Options
+
+   .. step:: Click the :guilabel:`Advanced` tab.
+
+      Select a :guilabel:`Read Preference` from the following options:
+
+      .. list-table::
+         :header-rows: 1
+         :widths: 50 50 
+
+         * - Read Preference
+           - Description
+
+         * - primary
+           - Default mode. All operations read from the current replica set primary.
+
+         * - primaryPreferred
+           - In most situations, operations read from the primary but if it is 
+             unavailable, operations read from secondary members. 
+
+         * - secondary
+           - All operations read from the secondary members of the replica set.
+
+         * - secondaryPreferred
+           - In most situations, operations read from secondary members but if 
+             no secondary members are available, operations read from the primary 
+             on sharded clusters.
+
+         * - nearest
+           - Operations read from a random eligible replica set member, 
+             irrespective of whether that member is a primary or secondary, based 
+             on a specified latency threshold. The operation considers the following 
+             when calculating latency:
+
+             - The localThresholdMS connection string option
+             - The maxStalenessSeconds read preference option
+             - Any specified tag sets 
+
+      For more information, see :manual:`Read Preferrence </core/read-preference/#read-preference>`.
```

Comment:
This could be a good candidate for a `seealso` directive. We could try having two bullets, one to the Read Preference page as currently provided, and one to [Read Preference Use Cases](https://www.mongodb.com/docs/manual/core/read-preference-use-cases/).

Reply:
Done.